osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -858,120 +897,30 @@ - (void)closeSplash {
-                                    initWithData:nsdataFromBase64String encoding:NSUTF8StringEncoding];
-        self.password = decodePassword;
-        self.confirmPassword = decodePassword;
-    }
-}
-
-
-- (void)resetPreferences {
-    NSString *blank = @"";
-    self.username = blank;
-    self.password = blank;
-
-    NSUserDefaults *prefs = [NSUserDefaults standardUserDefaults];
-    [prefs setObject:blank forKey:@"username"];
-    [prefs setObject:blank forKey:@"password"];
-    // REF: https://github.com/iwasrobbed/Objective-C-CheatSheet#storing-values
-    [prefs synchronize];
-    [self loadPreferences];
-}
-
+//- (void)loadPreferences {
+    // TODO(amodia): Set custom database path in the preferences menu.
+//}
 
 - (void)savePreferences {
     /*
-     1. Validate the following:
-        * username length max of 30 characters
-        * password length max of 128 characters
-        * username allowed characters are "letters, numbers and @/./+/-/_ characters" based on django.contrib.auth.models.AbstractUser
-     2. Save the preferences: REF: http://stackoverflow.com/questions/10148788/xcode-cocoa-app-preferences
-     3. Run `kalite manage setup` if no database was found.
+     1. Save the preferences: REF: http://stackoverflow.com/questions/10148788/xcode-cocoa-app-preferences
+     2. Run `kalite manage setup` if no database was found.
      */
     
-    NSString *username = self.stringUsername.stringValue;
-    NSString *password = self.stringPassword.stringValue;
-    NSString *confirmPassword = self.stringConfirmPassword.stringValue;
-    
-    self.username = username;
-    self.password = password;
-    self.confirmPassword = confirmPassword;
-    
-    if (self.processCounter != 0) {
-        alert(@"KA Lite is still processing, please wait until it is finished.");
-        return;
-    }
-
-    if (self.username == nil || [self.username isEqualToString:@""]) {
-        alert(@"Username must not be blank and can only contain letters, numbers and @/./+/-/_ characters.");
-        return;
-    }
-
-    NSString *usernameChars = getUsernameChars();
-    if ([self string:usernameChars containsAllCharactersInString:self.username] == NO) {
-        alert(@"Invalid username characters found, please use letters, numbers and @/./+/-/_ characters.");
-        return;
-    }
-    
-    if ([self.username length] > 30) {
-        alert(@"Username must not exceed 30 characters.");
-        return;
-    }
-
-    if (self.password == nil || [self.password isEqualToString:@""]) {
-        alert(@"Invalid password or the password does not match on both fields.");
-        return;
-    }
-
-    if (![self.password isEqualToString:self.confirmPassword]) {
-        alert(@"The password does not match on both fields.");
-        return;
-    }
-
-    if ([self.password length] > 128) {
-        alert(@"Password must not exceed 128 characters.");
-        return;
-    }
-    
     // Save the preferences.
```

Comment:
@amodia If you meant to re-instate these preferences-related codes as we discussed, I suggest you comment them out adding your remarks why you did so and what you intended to do.

Leaving them enabled (without doing anything) will be confusing during merges.